server/repository/mongorepo.py

```diff
@@ -41,27 +42,25 @@ def list(self, user=None, shelter=None):
         if shelter:
             db_filter["shelter"] = shelter
 
-        projection = {"_id": 0}
         user_shifts = [WorkShift.from_dict(i) for i in self.collection.find \
-                       (filter=db_filter, projection=projection)]
-
+                       (filter=db_filter)]
         return user_shifts
 
     def add(self, work_shift):
         """
         Add a WorkShift object to the data.
+        A unique id gets generated in mongoDB and is added to workShift object
         """
-        self.collection.insert_one(work_shift)
         work_shift.pop("_id")
+        self.collection.insert_one(work_shift)
```

Comment:
```suggestion
        # Remove the _id field from the dictionary, so that when we call insert_one
        # mongoDB will add an _id field to the work_shift dictionary object with a unique
        # value before inserting it into the collection. This ensures that each work shift
        # stored in our database has a unique _id
        work_shift.pop("_id")
        self.collection.insert_one(work_shift)
```